trunner/device.py

```diff
@@ -6,6 +6,7 @@
 
 from .config import PHRTOS_PROJECT_DIR
 
+TIMEOUT = 8
```

Comment:
add comment that it's default timeout for any input from the device (or even provide a better name)

Reply:
I changed name and added keyword 'timeout' to YAML config